src/api/models/article.js

```diff
@@ -0,0 +1,52 @@
+const Article = (sequelize, DataTypes) => {
+  const article = sequelize.define('Article',
+    {
+      id: {
+        allowNull: false,
+        unique: true,
```

Comment:
why UUID and others are using id

Reply:
I will have to update this model to @murediane 's model. I created it just to make sure that I finish my work. But as you are the one who created the articles users model, please, suggest me what I have to change